slc_ci_setup/cli/commands/deploy_source_credentials.py

```diff
@@ -0,0 +1,26 @@
+import click
+
+from slc_ci_setup.cli.cli import cli
+from slc_ci_setup.cli.common import add_options
+from slc_ci_setup.lib.run_deploy_source_credentials import run_deploy_source_credentials
+from slc_ci_setup.cli.options.aws_options import aws_options
+
+SOURCE_CREDENTIALS_STACK_NAME = "SLCSourceCredentials"
+
+@cli.command()
+@add_options(aws_options)
+@click.option('--secret-name', required=True, type=str,
+                 help="Secret stored in AWS Secret Manager.")
+@click.option('--secret-user-key', required=True, type=str,
```

Comment:
Why two keys?

Reply:
I added the user name also to AWS SecretsManager

Reply:
So it doesn't appear here as plain text in the repo and gives us more flexibility

Reply:
Ok that call it user-name and not user-key

Reply:
Hm, I am afraid that would to certain confusion. Secret entries in AWS SecretManager are identified by the Name of the secret (first parameter), and then a list of key/values. User key prameter here represents the key in the secret of the user name in that list.